src/System.Windows.Forms.DataVisualization/Utilities/XmlSerializer.cs

```diff
@@ -907,23 +907,23 @@ internal bool IsSerializableContent(string propertyName, object parent)
 			bool	serializable = true;
 			if(_serializableContent.Length > 0 || _nonSerializableContent.Length > 0)
 			{
-				int		serialzableClassFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
-				int		serialzablePropertyFitType = 0;	// 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
+                // 0 - undefined; 1 - '*'; 2 - 'Back*'; 3 - Exact
```

Comment:
Lost the meaning of the comments.

Reply:
This comments was for:
`serialzableClassFitType`
and
`serialzablePropertyFitType`